docs/src/pages/get-started/theming.mdx

```diff
@@ -0,0 +1,147 @@
+import ThemeObjectAttributes from '../../components/ThemeObjectAttributes'
+import ThemeLayout from '../../components/ThemeLayout'
+
+export default ThemeLayout
+
+### Theme Provider
+
+In order to style your app, you must wrap it in the `ThemeProvider` component.
+This component uses the `React.Context` API under the hood.
+
+In order to use the `ThemeProvider` component, you must pass it a value prop that is a `Theme` object.
+
+## Simple theming
+
+The `ThemeProvider` Component, you can either override the default styles(shown below) or use your own theme completely.
+You can add anything you want to the theme object, as long as you have required properties.
```

Comment:
I think we may want to caution users that theming is not 100% complete and some things may not work as expected – they should also be mindful of making sure the `theme` they provide to the ThemeProvider has all of the necessary theme functions included.

The way you show will work fairly well (`{ ...defaultTheme, ...myCustomTheme }`) but I think there are some unique cases that are looking specifically at `defaultTheme` instead of the context 😬

Reply:
Thanks for clarifying, I wasn't sure if theming was fully complete yet, will definitely add that little note